src/api/app/views/webui/users/tokens/edit.html.haml

```diff
@@ -6,22 +6,57 @@
 
     .row
       .col-12.col-md-10.col-lg-8
-        -# "as: :token" to have a consistent name across all various token classes
-        = form_for(@token, as: :token, url: token_path, method: :put, local: true) do |f|
-          .form-row#edit-token
+        = form_with(url: token_path, scope: :token, method: :put, local: true) do |f|
+          .form-row.d-none#created-token
+            .col-12.col-md-10.col-lg-9
+              %fieldset.form-group
+                = f.label(:string_readonly, 'Your new OBS Personal Access Token Secret', class: 'col-form-label col-form-label-lg')
```

Comment:
I think you should warn people that this is the only time they are going to see this...

Reply:
I added a simlar warning as it is displayed when creating a token, in the flash message.